src/components/CommentInfo.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Comment } from '../types/Comment';
+
+type Props = {
+  comment: Comment,
+  commentDestroyer: (id: number) => void,
```

Comment:
Consider renaming 'commentDestroyer' to 'onDeleteComment' for better readability and consistency with common naming conventions.